tests/tests_geomstats/test_geometry/data/discrete_curves.py

```diff
@@ -67,20 +56,21 @@ class SRVShapeBundleTestData(FiberBundleTestData):
     def tangent_vector_projections_orthogonality_with_metric_test_data(self):
         return self.generate_random_data()
 
-    def horizontal_geodesic_vec_test_data(self):
-        data = []
-        for n_times in [20]:
-            data.extend(
-                [dict(n_reps=n_reps, n_times=n_times) for n_reps in self.N_VEC_REPS]
-            )
-        return self.generate_tests(data)
 
-    def horizontal_geodesic_has_horizontal_derivative_test_data(self):
-        data = []
-        for n_times in [20]:
-            data.extend([dict(n_points=n_points, n_times=n_times) for n_points in [1]])
-        return self.generate_tests(data)
+class AlignerCmpTestData(TestData):
```

Comment:
What is "Cmp"? Is AlignedCmp a class in the codebase? Can you leave a docstring explaining why it deserves to have a spcial TestData?

Reply:
"Cmp" stands for comparison. This is a kind of test I've recently added that compares the outputs of two objects that do the same job. e.g. in this case it compares the outputs of the two algorithms we have for alignment.

I've implemented a similar comparison for metrics: if we have two different ways of implementing a given metric, it is very powerful to compare the results both provide as it would be very unlikely two independent ways of doing the same thing give the same results in random data.